site/docs/concepts/connectors.md

```diff
@@ -316,27 +316,15 @@ materializations:
       connector:
         image: ghcr.io/estuary/materialize-postgres:dev
         config:
-          # When using a proxy like SSH tunneling, set to localhost
-          host: localhost
-          # Specify an open port on your local machine to connect to the proxy.
-          port: 15432
+          address: 127.0.0.1:5432
```

Comment:
The `materialize-postgres` connector hasn't actually been updated to use `address` yet. But that's on my todo list for the near future, so I can throw that together tomorrow if it'll help to update the docs all at once.

Reply:
My bad - to make it easier for now I just changed the example to use `source-postgres`, since it really doesn't need to be one way or the other!